docker-bake.hcl

```diff
@@ -137,10 +137,10 @@ target "lint" {
   matrix = {
     buildtags = [
       { name = "default", tags = "", target = "golangci-lint" },
-      { name = "labs", tags = "dfrunsecurity dfparents", target = "golangci-lint" },
-      { name = "nydus", tags = "nydus", target = "golangci-lint" },
-      { name = "yaml", tags = "", target = "yamllint" },
-      { name = "proto", tags = "", target = "protolint" },
+      //{ name = "labs", tags = "dfrunsecurity dfparents", target = "golangci-lint" },
+      //{ name = "nydus", tags = "nydus", target = "golangci-lint" },
+      //{ name = "yaml", tags = "", target = "yamllint" },
+      //{ name = "proto", tags = "", target = "protolint" },
```

Comment:
Can be reverted since https://github.com/moby/buildkit/pull/4490

Reply:
Removed. This was for testing and was accidentally committed.